CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 * Enhancements:
   * Corrected and enhanced static type hints for functions and methods that make
     CMR queries or handle CMR query results (#508)
+  * Enable queries to Earthdata test environments for authenticated accounts (pull #426)
```

Comment:
Changed!
I didn't change other lines in the changelog, e.g., the other two lines in the "unreleased" section, because I'm not totally sure which issues they correspond to.

Reply:
@chuckwondo, would you like me (or would you like) to create a new _issue_: for updating the changelog to have links throughout?

Reply:
Good idea. I'll create a new issue for it.